README.md

```diff
@@ -130,7 +130,7 @@ You can also selectively apply this plugin in a single subproject if desired.
 
 The project needs to be imported as a gradle project into your IDE.
 
-Then you should be able to open [`RideCleansingTest`](ride-cleansing/src/test/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingTest.java) and run this test.
+Then you should be able to open [`RideCleansingUnitTest`](ride-cleansing/src/test/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingUnitTest.java) and run this test.
```

Comment:
Sorry, what was the reason of touching docs?

Reply:
because there is no RideCleansingTest.java in the project and there is RideCleansingUnitTest.java which should be refered there